lib/rules/web/ocm_console/cluster_detail.xyaml

```diff
@@ -702,3 +702,21 @@ subscription_settings_error_message_loaded:
         selector:
             xpath: //div[contains(., '<error_message>')]
         timeout: 30
+wait_cluster_addon_uninstallation_on_detail_page:
+    element:
+        selector:
+            xpath: //div[@class='pf-l-grid pf-m-gutter']//span[text()='Add-on uninstallation']/../../dd/div[text()='<cluster_status>']
+        timeout: <wait_time>
+wait_cluster_uninstallation_on_detail_page:
+    element:
+        selector:
+            xpath: //div[@class='pf-l-grid pf-m-gutter']//span[text()='Cluster uninstallation']/../../dd/div[text()='<cluster_status>']
+        timeout: <wait_time>
+wait_for_uninstallation_to_finish:
+    elements:
+        - selector:
+            xpath: //h1[text()='Clusters']
+          timeout: 2000
+        - selector:
+            xpath: //button[contains(.,'Name')]
+          timeout: 2000
```

Comment:
What is the exit condition for this rule `wait_for_uninstallation_to_finish`? And is the timeout time `2000` reasonable, @xueli181114 ?